algo/src/main/java/com/hartwig/serve/ServeConfig.java

```diff
@@ -250,6 +265,9 @@ static ServeConfig createConfig(@NotNull CommandLine cmd) throws ParseException
                 .ckbBlacklistEvidenceTsv(useCkbEvidence ? nonOptionalFile(cmd, CKB_BLACKLIST_EVIDENCE_TSV) : NOT_APPLICABLE)
                 .ckbDrugCurationTsv(useCkbEvidence ? nonOptionalFile(cmd, CKB_DRUG_CURATION_TSV) : NOT_APPLICABLE)
                 .ckbRegionTsv(useCkbTrials ? nonOptionalFile(cmd, CKB_REGION_TSV) : NOT_APPLICABLE)
+                .ckbFacilityCurationNameTsv(useCkbTrials ? nonOptionalFile(cmd, CKB_FACILITY_CURATION_NAME_TSV) : NOT_APPLICABLE)
+                .ckbFacilityCurationZipTsv(useCkbTrials ? nonOptionalFile(cmd, CKB_FACILITY_CURATION_ZIP_TSV) : NOT_APPLICABLE)
+                .ckbFacilityCurationManualTsv(useCkbTrials ? nonOptionalFile(cmd, CKB_FACILITY_CURATION_MANUAL_TSV) : NOT_APPLICABLE)
```

Comment:
This is tricky because in ServeAlgo you use these params also if only `useCkbEvidence` is true. 

I think I would check for both (`useCkbTrials || useCkbEvidence`) in this config